conjur/wrapper/http_wrapper.py

```diff
@@ -55,11 +53,19 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
     request_method = getattr(requests, http_verb.name.lower())
 
     #pylint: disable=not-callable
-    response = request_method(url, *args,
-                              params=query,
-                              verify=ssl_verify,
-                              auth=auth,
-                              headers=headers)
+    try:
+        response = request_method(url, *args,
+                                  params=query,
+                                  verify=True,
+                                  auth=auth,
+                                  headers=headers)
+    except requests.exceptions.SSLError:
```

Comment:
this is the exception for the wrong cert I guess.. are there anymore exceptions that could be caused by wrong cert?

Reply:
if self-signed cert and verify=True the following error will be raised:
`'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain`